tests/automation_framework/tests/worker_tests/test_update.py

```diff
@@ -108,3 +108,29 @@ def test_worker_update_invalid_parameter(self):
 
         logger.info('\t\t!!! Test completed !!!\n\n')
 
+    @pytest.mark.worker
```

Comment:
Add this to the bug that I have filed on worker update cases not able to handle negative scenarios: https://jira.devtools.intel.com/browse/TCFH-788

Reply:
Done